packages/paste-website/src/pages/icon-system/index.mdx

```diff
@@ -0,0 +1,235 @@
+---
+title: Icon Usage Guidelines
+description: Overview on how to use icons in the context of Paste projects.
+---
+
+import {graphql} from 'gatsby';
+import {Link} from 'gatsby';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+import {Anchor} from '@twilio-paste/anchor';
+import {Button} from '@twilio-paste/button';
+import {PlusIcon} from '@twilio-paste/icons/esm/PlusIcon';
+import {LoadingIcon} from '@twilio-paste/icons/esm/LoadingIcon';
+import {Callout, CalloutTitle, CalloutText} from '../../components/callout';
+import {DoDont, Do, Dont} from '../../components/DoDont';
+import Changelog from '@twilio-paste/icons/CHANGELOG.md';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/icon-system/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+<Callout>
+  <CalloutText>
+    You can view and play with icons on <Anchor href="https://twilio-labs.github.io/paste/?path=/story/overview-icons--list">our storybook icon list</Anchor>.
+    Our icon set is still limited, more will be adding over time. Please{' '}
+    <Anchor href="https://github.com/twilio-labs/paste/issues">file an icon request</Anchor> if you need an icon we
+    don't yet have. 
+  </CalloutText>
+</Callout>
+
+
+Icons are small graphical representation of a program or a function. They can be used to enhance the
+appearance of a web interface and break up the monotony of text. 
+
+Use an icon only when it provides additional clarity or is necessary to draw attention to a UI element.
+If an icon is used to indicate hierarchy of a UI element, all elements at the same hierarchy should also be paired with an icon.
+
+
+## Accessibility
+
+### Specify if an icon is decorative
+
+Icons can either be **decorative** or **informative**. *"Decorative icons don’t add information to the content 
+of a page. The information provided by the image might already be given using adjacent text, or the image 
+might be included to make the website more visually attractive."*<sup><Anchor href="https://www.w3.org/WAI/tutorials/images/decorative/">w3.org</Anchor></sup> 
```

Comment:
The w3 link doesn't need to be superscript. It collides with the quotation mark. You could just say:

> From w3.org: "Decorative icons..."

or

> "Decorative icons...visually attractive" (w3.org).